portfolio/src/main/webapp/index.html

```diff
@@ -16,125 +16,144 @@
-	var angle = Math.random()*180;
-	function update()
-	{
-
-		for(var i = 0; i < mp; i++)
-		{
-            
-			var p = particles[i];
-			//Updating X and Y coordinates
-			//We will add 1 to the cos function to prevent negative values which will lead flakes to move upwards
-			//Every particle has its own density which can be used to make the downward movement different for each flake
-			//Lets make it more random by adding in the radius
-			p.y += ((-((Math.cos(angle) + 1)))*(p.r-.55))/2;
-			p.x += ((((Math.cos(angle) + 1)))*(p.r-.55))/2;
-			
-			//Sending flakes back from the top when it exits
-			//Lets make it a bit more organic and let flakes enter from the left and right also.
-			if(p.x > W+5 || p.x < -5 || p.y < 0)
-			{
-				if(i%3 > 0) //66.67% of the flakes
-				{
-					particles[i] = {x: Math.random()*W, y: H, r: p.r, d: p.d};
-				}
-				else
-				{
-					//If the flake is exitting from the right
-					if(Math.sin(angle) > 0)
-					{
-						//Enter from the left
-						particles[i] = {x: -5, y: Math.random()*H, r: p.r, d: p.d};
-					}
-					else
-					{
-						//Enter from the right
-						particles[i] = {x: W+5, y: Math.random()*H, r: p.r, d: p.d};
-					}
-				}
-			}
-		}
-	}
-	
-	//animation loop
-	setInterval(draw, 33);
+		// Will it be a storm or peaceful?
+    var COUNT = 666;
+
+    // Get our cotaniner
+    var snowContainer = document.getElementById('starContainer');
+
+    // Create the canvas element
+    var canvas = document.getElementById('starCanvas');
+    var ctx = canvas.getContext('2d');
+
+    // Get the size of the container, that's why we defined the height in the HTML
+    var width = snowContainer.clientWidth;
+    var height = snowContainer.clientHeight;
+    var i = 0;
+    var active = false;
+
+    function onResize() {
+        width = snowContainer.clientWidth;
+        height = snowContainer.clientHeight;
+        canvas.width = width;
+        canvas.height = height;
+        ctx.fillStyle = '#FFF';
+
+        requestAnimFrame(update);
+    }
+
+    var Snowflake = function () {
+        this.x = Math.random() * width;
+        this.y = Math.random() * height;
+        this.r = Math.random() + .55;
+        this.vy = this.r * ((.1 + Math.abs(Math.random() - .5)) * -.28);
+        this.vx = this.vy;
+
+        // Bigger snow?
+        this.o = 0.5 + Math.random();
+    }
+
+    // You can set up the
+    Snowflake.prototype.reset = function () {
+        if (Math.random() > 0.5) {
+            this.x = width;
+            this.y = Math.random() * height;
+        } else {
+            this.x = Math.random() * width;
+            this.y = height;
+        }
+
+        // More speed? Change this
+
+        this.r = Math.random() + .55;
+        this.vy = this.r * (Math.abs(Math.random() - 1) * -.23);
+        this.vx = this.vy;
+
+        // Bigger snow?
+        this.o = 0.5 + Math.random();
```

Comment:
I think this is only used for "globalAlpha" which is transparency of the snow, maybe make this more readable by changing it to "this.transparency"? 
Also I think the comment above would be wrong as it's not bigger snow, just less transparent snow.  Let me know if I'm wrong though! (And same for line 87)